yt/data_objects/static_output.py

```diff
@@ -339,6 +343,20 @@ def force_periodicity(self, val=True):
             raise TypeError("force_periodicity expected a boolean.")
         self._force_periodicity = val
 
+    def set_periodicity(self, val: Union[bool, Tuple[bool, bool, bool]]) -> None:
```

Comment:
round 2: this is in direct competition with `force_periodicity`. Since it's much more general, I suggest refactoring around this new method:
- get rid of the `_force_periodicity` attr
- make the `force_periodicity` method call `set_periodicity` internally.

Reply:
Definitely, let me fix this.